common-ui/build.gradle

```diff
@@ -17,14 +17,22 @@
 plugins {
     id 'com.android.library'
     id 'kotlin-android'
-    id 'kotlin-kapt'
+    id 'com.squareup.anvil'
 }
 
 apply from: "$rootProject.projectDir/gradle/android-library.gradle"
 
 dependencies {
 
+    implementation project(path: ':di')
+
+    implementation Google.dagger
+
+    implementation Kotlin.stdlib.jdk7
     implementation AndroidX.appCompat
     implementation Google.android.material
     implementation AndroidX.constraintLayout
+
```

Comment:
Just double-checking, are all these changes related to this refactor? no need to explain why we use each library, just checking we are not including something not related.

Reply:
yes, we need to add AppCompat to reference AppCompatActivity, etc...